taskcluster/__init__.py

```diff
@@ -14,6 +11,3 @@
     if len(log.handlers) == 0:
         log.addHandler(logging.StreamHandler())
 log.addHandler(logging.NullHandler())
-
-from client import *  # NOQA
```

Comment:
That's correct, if you want to use stuff in taskcluster.client, that'll be `import taskcluster.client` or `from taskcluster.client ...`